config/i18n/all-langs.ts

```diff
@@ -45,7 +45,8 @@ export const auditedCerts = {
     SuperBlocks.JsAlgoDataStruct,
     SuperBlocks.FrontEndDevLibs,
     SuperBlocks.DataVis,
-    SuperBlocks.BackEndDevApis
+    SuperBlocks.BackEndDevApis,
+    SuperBlocks.QualityAssurance,
```

Comment:
```suggestion
    SuperBlocks.QualityAssurance
```
I think prettier is complaining about the trailing comma.

Reply:
Sorry, missed it.